File: example_bots/java/ChickenBot.java
/**
 * A bot that always chickens out
 *
 * This bot checks if the current call size is greater than 0,
 * that is, it checks if another player has raised the pot.
 * If that is the case, it chickens out and folds!
 */
public class ChickenBot {
    public static final String BOT_NAME = "Chicken bot";

    public static int act(Observable obs) throws Exception {
        int amountToCall = obs.getCallSize();
        if (amountToCall > 0) {
            return 0;
        } else {
            return 1;
        }
    }
}

File: example_bots/python/chicken_bot.py
from poker_game_runner.state import Observation
from poker_game_runner.utils import Range, HandType


class Bot:
    def get_name(self):
        return "chicken_bot"

    def act(self, obs: Observation):
        # This bot checks if the amount to call is greater than 0
        # that is, it checks if anyone has raised.
        # If that is the case, it chickens out and folds!

        amount_to_call = obs.get_call_size()
        if amount_to_call > 0:
            return 0
        else:
            return 1
